### PR TITLE
Chore/environment setup

### PR DESCRIPTION
**Pull request by PascalUlor**
#### What does this PR do?
```
Setup development environment
create the database
setup server
seed dummy users and surveys
Query database
```
#### How should this be manually tested?

clone this repo
create a Postgres database on pgAdmin
create a .env file with the following format
`DEV_DATABASE_URL='postgres://<dbusername>:<dbpassword>@localhost:5432/<dbname>'`
run server (npm run dev)
in browser enter the url
`http://localhost:8000/graphql`

open localhost on any browser

#### Any background context you want to provide?
```
Chrome is a preferred browser
```

### CODE REVIEW

File: database/mock/user.mock.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  // {
+  //   username: 'Pascal Ulor',
+  //   email: 'pascal@.com.au',
+  //   password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+  //   createdAt: new Date(),
+  //   updatedAt: new Date(),
+  // },
```
Comment (Easybuoy) [inline database/mock/user.mock.js]: If there is no need for this, i think you can delete it.

Comment (Easybuoy) [reply database/mock/user.mock.js]: Everything else looks good. Nice work guys 🔥

Comment (PascalUlor) [reply database/mock/user.mock.js]: we needed to some seed data to test the database

**Comment by PascalUlor** [comment]
[Set up database](https://trello.com/c/Egy2ZrUJ/6-set-up-database)

**Pull request merged**